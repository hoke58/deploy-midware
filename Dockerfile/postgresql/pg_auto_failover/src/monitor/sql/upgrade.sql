-- Copyright (c) Microsoft Corporation. All rights reserved.
-- Licensed under the PostgreSQL License.

CREATE EXTENSION pgautofailover VERSION '1.0';
\dx pgautofailover

ALTER EXTENSION pgautofailover UPDATE TO '1.1';
\dx pgautofailover

ALTER EXTENSION pgautofailover UPDATE TO '1.2';
\dx pgautofailover

ALTER EXTENSION pgautofailover UPDATE TO '1.3';
\dx pgautofailover

DROP EXTENSION pgautofailover;
