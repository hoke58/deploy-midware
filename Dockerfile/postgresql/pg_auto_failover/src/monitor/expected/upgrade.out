-- Copyright (c) Microsoft Corporation. All rights reserved.
-- Licensed under the PostgreSQL License.
CREATE EXTENSION pgautofailover VERSION '1.0';
\dx pgautofailover
             List of installed extensions
      Name      | Version | Schema |   Description    
----------------+---------+--------+------------------
 pgautofailover | 1.0     | public | pg_auto_failover
(1 row)

ALTER EXTENSION pgautofailover UPDATE TO '1.1';
\dx pgautofailover
             List of installed extensions
      Name      | Version | Schema |   Description    
----------------+---------+--------+------------------
 pgautofailover | 1.1     | public | pg_auto_failover
(1 row)

ALTER EXTENSION pgautofailover UPDATE TO '1.2';
\dx pgautofailover
             List of installed extensions
      Name      | Version | Schema |   Description    
----------------+---------+--------+------------------
 pgautofailover | 1.2     | public | pg_auto_failover
(1 row)

ALTER EXTENSION pgautofailover UPDATE TO '1.3';
\dx pgautofailover
             List of installed extensions
      Name      | Version | Schema |   Description    
----------------+---------+--------+------------------
 pgautofailover | 1.3     | public | pg_auto_failover
(1 row)

DROP EXTENSION pgautofailover;
